dali/base/dasess.cpp

```diff
@@ -2055,36 +2066,56 @@ class CUserDescriptor: implements IUserDescriptor, public CInterface
         passwordenc.clear();
         sessionToken = 0;
         signature.clear();
+        utcTimeStamp.clear();
+        userTimeStampSignature.clear();
     }
     void serialize(MemoryBuffer &mb)
     {
         mb.append(username).append(passwordenc);
     }
-    void serializeExtra(MemoryBuffer &mb)
+    void serializeSignature(MemoryBuffer &mb)
     {
         if (queryDaliServerVersion().compare("3.14") >= 0)
```

Comment:
I bumped up to 3.14 during Master (7.0) development, so this will not be a problem. Also, note that this only would be a problem if someone had installed public/private keys and enabled signatures, which so far has only been me

Reply:
3.15 it shall be